eng/scripts/Language-Settings.ps1

```diff
@@ -244,7 +244,7 @@ function ValidatePackagesForDocs($packages) {
     # Get value for variables outside of the Foreach-Object scope
     $scriptRoot = "$using:scriptRoot"
     $workingDirectory = "$using:tempDirectory"
-    return ."$scriptRoot\validate-docs-package.ps1" -Package $_ -WorkingDirectory $workingDirectory
+    return ."$scriptRoot\validate-docs-package.ps1" -Package $_ -DocValidationImageId "$using:imageId" -WorkDirectory $workingDirectory 
```

Comment:
Where is `$using:imageId` value coming from?

Reply:
LanguageSetting.ps1 has ImageId as parameter. I use the parameter over there.

Reply:
I made it wrong. Actually, it is Update-DocsMsPackage.ps1 pass it through. It will default to empty if nothing passed in.